src/hotspot/os/aix/os_aix.cpp

```diff
@@ -1296,6 +1296,9 @@ void os::print_memory_info(outputStream* st) {
   vmembk_print_on(st);
 }
 
+// Unimplemented
```

Comment:
Is this temporary or does AIX not support a way to get RSS?

Reply:
When I was still more deeply involved, I did not find a way. There may be one now; we also could re-use NMT commit total for that, if NMT is enabled. So, I am not sure.